packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Toggle from 'terra-toggle';
+import { injectIntl, intlShape } from 'terra-base';
+import IconChevronDown from 'terra-icon/lib/icon/IconChevronDown';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './ShowHide.module.scss';
+import Button from './_ShowHideButton';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content in the body of the component that will be shown or hidden
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Text that will be visible to the user while the component is collapsed
+   */
+  preview: PropTypes.string.isRequired,
+  /**
+   * Button text that will be displayed when the component is collapsed
+   */
+  collapsedButtonText: PropTypes.string,
+  /**
+   * Button text that will be displayed when the component is expanded
+   */
+  expandedButtonText: PropTypes.string,
+  /**
+   * Icon displayed next to the button text
+   */
+  icon: PropTypes.element,
+  /**
+   * The intl object to be injected for translations. Provided by the injectIntl function.
+   */
+  intl: intlShape.isRequired,
+  /**
+   * Sets the animation for the component when it is expanded or collapsed
+   */
+  isAnimated: PropTypes.bool,
+  /**
+   * Sets the initial state of the component to open and expanded
+   */
+  isInitiallyOpen: PropTypes.bool,
+  /**
+   * Callback function triggered when the component is collapsed
+   */
+  onClose: PropTypes.func,
+  /**
+   * Callback function triggered when the component is expanded
+   */
+  onOpen: PropTypes.func,
+};
+
+const defaultProps = {
+  icon: <IconChevronDown />,
+  isAnimated: false,
+  isInitiallyOpen: false,
+};
+
+const animatedStyle = {
+  // fix for the toggle oscillation when setting isAnimated to true
+  transitionDelay: '0.3s',
+  WebkitTransitionDelay: '0.3s', /* Safari */
+};
```

Comment:
Does this work in IE10? Can we do the transitions in CSS?

Reply:
Here is an example of using a class. https://github.com/cerner/terra-core/blob/e9f0aad0f345ea10030211a477aa130c4191a3c4/packages/terra-toggle/src/Toggle.module.scss#L14

Reply:
which css attribute/selector would I use for ``` .toggle[] ```? 

``` .toggle[aria-hidden='false']``` works because aria-hidden is a valid selector. I don't see one for animated

Reply:
The class name can differ, but the style can be applied via stylesheets once without adding the webkit extension

Reply:
So if I'm understanding you correctly, instead of having ``` if (this.props.isAnimated) ```, I would just add an additional class to the class name declaration? Like this: ```
if (this.props.isAnimated) {
const showHideClassName = cx([
      'button',
      { 'is-open': this.state.isOpen },
      { 'animated': this.props.isAnimated },
      customProps.className,
    ]);```

Reply:
@emilyrohrbough helped me with the refactor for the styling as a CSS class: https://github.com/cerner/terra-core/pull/1910/commits/6979d941a0a0749df9f9ffeae82def7c0b9f34c5
I still need to test it in IE10

Reply:
I can confirm that the transition-delay styling works in IE10. I used the latest deployment to test: https://terra-core-deployed-pr-1910.herokuapp.com/#/components/terra-show-hide/show-hide/show-hide

And here is a screenshot of the Windows 7 virtual machine I am running through VirtualBox to test IE10: 
![image](https://user-images.githubusercontent.com/42552940/45979438-b8f8eb80-c014-11e8-862b-667c940f98b9.png)